src/options/options.ts

```diff
@@ -48,3 +49,20 @@ export function makeOptions(options: ProvidedCodeGenOptions): CodeGenOptions {
     ...options
   };
 }
+
+/**
+ * Validate that the options have required variables for custom generation.
+ */
+export function validOptions(options: ProvidedCodeGenOptions): void {
```

Comment:
>validOptions [](start = 16, length = 12)

Lets rename it to validateOptions to make it slightly more descriptive

Reply:
Yes, will do that! 

It would be nice if this was more built into to the options itself, I am just not sure how to implement that in a nice way.

Reply:
I have renamed it.

Reply:
Thanks!